### PR TITLE
Add user stories 7,8,9,10,11. I might be forgeting something I did, b…

### PR DESCRIPTION
**Pull request by borisovkaloyan**
…ut oh well

### CODE REVIEW

File: Code/Project/Project/FileWritePage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".txt";
+                StreamWriter writer = new StreamWriter(path);
+
+                writer.WriteLine(DateTime.Now.ToString());
+                writer.WriteLine();
+                writer.WriteLine("Project: " + Global.ProjectName);
+                writer.WriteLine("SCRUM master: " + Global.ScrumMasterName);
+                writer.WriteLine("==================================");
+
+                foreach (Task task in Global.Tasks)
```
Comment (amilisov) [inline Code/Project/Project/FileWritePage.xaml.cs:65]: Global tasks?

File: Code/Project/Project/FileWritePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+        private void JsonExport(object sender, RoutedEventArgs e)
+        {
+
+            var dialog = new FolderBrowserDialog();
+            DialogResult result = dialog.ShowDialog();
+
+            if (result == DialogResult.OK)
+            {
+                string path = dialog.SelectedPath + "\\SCRUM_Session_" + DateTime.Now.ToString("dd/MM/yyyy") + ".json";
```
Comment (amilisov) [inline Code/Project/Project/FileWritePage.xaml.cs]: The name would be better to include the date/time created AND the name of the project.

File: Code/Project/Project/FileWritePage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        private string scrumMasterName = "";
+        private string currentDate = "";
+        private int taskCount;
+        private List<Task> tasks = new List<Task>();
+
+        public string ProjectName { get { return projectName; } set { projectName = value; } }
+        public string ScrumMasterName { get { return scrumMasterName; } set {  scrumMasterName = value; } }
+        public string CurrentDate { get {  return currentDate; } set {  currentDate = value; } }
+        public int TaskCount { get {  return taskCount; } set {  taskCount = value; } }
+        public List<Task> Tasks { get {  return tasks; } set { tasks = value; } }
```
Comment (amilisov) [inline Code/Project/Project/FileWritePage.xaml.cs]: You dont need the private versions of the vars. "pulbic Type Var" would be enough.

File: Code/Project/Project/Task.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+{
+    public class Task
+    {
+        private string _Name = "";
+        private string _Description = "";
+        private int _Rating;
+
+        public string Name { get { return _Name; } set { _Name = value; } }
+        public string Description { get { return _Description; } set { _Description = value; } }
+        public int Rating { get { return _Rating; } set { _Rating = value; } }
```
Comment (amilisov) [inline Code/Project/Project/Task.cs]: You dont need the private versions of the vars. "pulbic Type Var" would be enough.

File: Code/Project/Project/FileWritePage.xaml.cs (hunk 1 above)
Comment (borisovkaloyan) [reply Code/Project/Project/FileWritePage.xaml.cs:65]: yes. it's easier

File: Code/Project/Project/FileWritePage.xaml.cs (hunk 2 above)
Comment (borisovkaloyan) [reply Code/Project/Project/FileWritePage.xaml.cs]: sure

File: Code/Project/Project/FileWritePage.xaml.cs (hunk 3 above)
Comment (borisovkaloyan) [reply Code/Project/Project/FileWritePage.xaml.cs]: sure

File: Code/Project/Project/Task.cs (hunk 4 above)
Comment (borisovkaloyan) [reply Code/Project/Project/Task.cs]: sure

**Pull request merged**